lib/app/modules/ImportExport/controllers/import_controller.dart

```diff
@@ -111,31 +110,20 @@ class ImportController extends GetxController
   }
 
   void pickImportFile() async {
-    try {
-      final importPath = await FlutterFileDialog.pickFile(
-        params: const OpenFileDialogParams(
-          fileExtensionsFilter: ['json'],
-          mimeTypesFilter: ['application/json'],
-        ),
-      );
-
-      if (importPath == null) {
-        return;
-      }
-
-      final js = json.decode(await File(importPath).readAsString())
-          as Map<String, dynamic>;
-      toImport.value = ImportSelections.fromJson(js);
-    } catch (e) {
-      // unawaited(analytics.logEvent(name: Events.ImportFail, parameters: {
-      //   'reason': e.toString(),
-      // }));
+    var result =
+        await FilePicker.platform.pickFiles(type: FileType.custom, allowedExtensions: ['json', 'dungeonpaper']);
```

Comment:
Is `.dungeonpaper` a relevant extension? I think we only need `json` here
```suggestion
        await FilePicker.platform.pickFiles(type: FileType.custom, allowedExtensions: ['json']);
```

Reply:
No, but I feel in a future change it should be. Lots of things make `json` files, and ideally you wouldn't have to confuse those with valid files.

However, this change should happen alongside an export change.